src/Ant/Checkbox/Css.elm

```diff
@@ -26,14 +34,33 @@ emptyContent : Style
 emptyContent =
     property "content" "\"\""
 
+withFocus : String -> String
+withFocus class = class ++ ":focus"
+
+withHover : String -> String
+withHover class = class ++ ":hover"
+
+withChecked: String -> String
+withChecked class = class ++ "-checked"
+
+withAfter : String -> String
+withAfter class = class ++ "::after"
```

Comment:
`Hover`, `focus` and `after` are supported through elm-css. Please rely on the elm-css api first, and write raw css only as a fallback. 

https://package.elm-lang.org/packages/rtfeldman/elm-css/latest/Css#after

Reply:
This is nice ! However it doesn't work if you are the :hover is not on the element but a parent for instance